pkg/scheduler/plugins/elasticquota/core/runtime_quota_calculator.go

```diff
@@ -386,7 +386,7 @@ func (qtw *QuotaTreeWrapper) SetClusterTotalResource(full v1.ResourceList) {
 }
 
 // UpdateOneGroupRuntimeQuota update the quotaInfo's runtimeQuota as the quotaNode's runtime.
-func (qtw *QuotaTreeWrapper) UpdateOneGroupRuntimeQuota(quotaInfo *QuotaInfo) {
+func (qtw *RuntimeQuotaCalculator) UpdateOneGroupRuntimeQuota(quotaInfo *QuotaInfo) {
```

Comment:
I think update request\min\max\sharedRatio no need to calculate every quota's runtime, only ++globalVersion;  when UpdateOneGroupRuntimeQuota,  if we find quota's ver not equal to global version, then execute adjustQuotaNoLock(rename to calculateRuntimeQuotaNoLock),   and get real runtimeQuota.

Reply:
fix